src/remoted/manager.c

```diff
@@ -88,17 +90,35 @@ void cleaner(void* data){
  */
 void save_controlmsg(const keyentry * key, char *r_msg, size_t msg_length)
 {
-    char msg_ack[OS_FLSIZE + 1];
-    char *end;
-    char *uname = "";
-    pending_data_t *data;
-    FILE * fp;
-    mode_t oldmask;
+    char msg_ack[OS_FLSIZE + 1] = "";
+    char *msg = NULL;
```

Comment:
Should we group all these variables into a struct?

Reply:
FIXED